src/util.js

```diff
@@ -40,7 +40,22 @@ var isValue = c3_chart_internal_fn.isValue = function (v) {
     },
     getPathBox = c3_chart_internal_fn.getPathBox = function (path) {
         var box = path.getBoundingClientRect(),
-            items = [path.pathSegList.getItem(0), path.pathSegList.getItem(1)],
-            minX = items[0].x, minY = Math.min(items[0].y, items[1].y);
-        return {x: minX, y: minY, width: box.width, height: box.height};
+            minX, minY;
+        // MSIE supports pathSegList while it crashes on latest Chrome: https://msdn.microsoft.com/en-us/library/ff971976(v=vs.85).aspx
+        // See also: https://github.com/masayuki0812/c3/issues/1465
+        if (path.pathSegList && path.pathSegList.getItem) {
+            var seg0 = path.pathSegList.getItem(0);
+            var seg1 = path.pathSegList.getItem(1);
+            minX = Math.min(seg0.x, seg1.x); 
+            minY = Math.min(seg0.y, seg1.y);
+        } else {
+            minX = box.left;
```

Comment:
Testing this with a breakpoint at line 1364 inside the `isWithinBar` function, on [one of the demo charts](http://c3js.org/samples/chart_combination.html) it appears that `box.left` is not the same as `seg0.x`.

![screen shot 2016-02-16 at 11 35 31 am](https://cloud.githubusercontent.com/assets/214379/13088524/6b494a30-d4a1-11e5-97e3-c33beccba9b7.png)

I've seen some people [subtract the svg container's `boundingClientRect` from the element's](http://stackoverflow.com/a/18554375) but I'm not sure if that's what we want here because those results aren't exactly equal to the `pathSegList` values either.

![screen shot 2016-02-16 at 11 42 02 am](https://cloud.githubusercontent.com/assets/214379/13088744/57b92e62-d4a2-11e5-89ba-d7243a6dc821.png)